projects/assetchain-swap/index.js

```diff
@@ -0,0 +1,10 @@
+const { uniV3Export } = require('../helper/uniswapV3')
+const ADDRESSES = require('../helper/coreAssets.json')
+
+
+module.exports = {
+    methodology: "Tracks the total liquidity in all Uniswap V3 pools",
+    assetchain: {
```

Comment:
also, this looks wrong, think the file you want to update is https://github.com/theiceeman/DefiLlama-Adapters/blob/pr/theiceeman/12156/projects/helper/tokenMapping.js#L150